Toolset/libraries/revidelibrary.8.livecodescript

```diff
@@ -3874,13 +3874,21 @@ function revIDEExtensionFolders
    return tFoldersA
 end revIDEExtensionFolders
 
-function revIDEDefaultExtensionIcon pType
+function revIDEDefaultExtensionIcon pType, pRetina
    switch pType
       case "library"
-         return revIDESpecialFolderPath("images") & slash & "extension-library.png"
+         if pRetina is true then
+            return revIDESpecialFolderPath("images") & slash & "extension-library@extra-high.png"
+         else
+            return revIDESpecialFolderPath("images") & slash & "extension-library.png"
+         end if
       case "widget"
       default
-         return revIDESpecialFolderPath("images") & slash & "extension-widget.png"
+         if pRetina is true then
+            return revIDESpecialFolderPath("images") & slash & "extension-widget@extra-high.png"
+         else
+            return revIDESpecialFolderPath("images") & slash & "extension-widget.png"
+         end if
    end switch
```

Comment:
I'm not 100% sure about this. The engine auto selects if a platform is retina and seeks the appropriate @size.. Should we be checking for retina case here? would setting the image path to "extension-widget.png" suffice?